src/libs/antares/study/area/scratchpad.cpp

```diff
@@ -157,5 +157,7 @@ AreaScratchpad::AreaScratchpad(const StudyRuntimeInfos& rinfos, Area& area) : ts
     pumpHasMod = (valuePumping > 0.);
 }
 
+AreaScratchpad::~AreaScratchpad() = default;
```

Comment:
We should remove the destructor here, and make it a default one in the corresponding **.h** source file